Makefile

```diff
@@ -90,9 +90,6 @@ EXPERIMENTAL_MANIFEST := ./manifests/experimental.yaml
 EXPERIMENTAL_E2E_MANIFEST := ./manifests/experimental-e2e.yaml
 CATALOGS_MANIFEST := ./manifests/default-catalogs.yaml
 
-# Manifest used by kind-deploy, which may be overridden by other targets
-SOURCE_MANIFEST := $(STANDARD_MANIFEST)
```

Comment:
Removing this is likely breaking things in multiple places (there are a number of failures)